schema/artifactIndex.d.ts

```diff
@@ -0,0 +1,33 @@
+export interface artifactPaths {
+    "/output/job/{uuid}/artifacts/{node-index}/{path}": {
+        get: artifactOperations['getArtifact']
+    }
+}
+
+export interface artifactOperations {
+    getArtifact: {
+        parameters: {
+            query?: never
+            header?: never
+            path?: {
+                uuid: string
```

Comment:
`uuid` here is a bit problematic as I have no idea what this ID is tied to. Seems like it changes across jobs but it isn't a job UUID. As far as I know there's no easy way to get this uuid other than retrieving list of artifacts and parsing it from a URL returned (which is really ugly).

Alternative would be to just pass the full path instead of building it like this but it would still require parsing response. 

(For reference Circle's API returns node index, path to file and full URL to artifact: https://circleci.com/docs/api/v2/index.html#operation/getJobArtifacts)